fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -104,11 +110,20 @@ public Table getTable(String dbName, String tblName) {
         }
 
         try {
-            org.apache.iceberg.Table icebergTable = icebergCatalog.loadTable(identifier);
-            Table table = IcebergApiConverter.toIcebergTable(
-                    icebergTable, catalogName, dbName, tblName, icebergCatalog.getIcebergCatalogType().name());
-            tables.put(identifier, table);
-            return table;
+            if (icebergCatalog instanceof IcebergHiveCatalog || icebergCatalog instanceof IcebergGlueCatalog) {
+                IcebergMetricsReporter metricsReporter = new IcebergMetricsReporter();
+                org.apache.iceberg.Table icebergTable = icebergCatalog.loadTable(identifier, metricsReporter);
+                Table table = IcebergApiConverter.toIcebergTable(icebergTable, catalogName, dbName, tblName,
```

Comment:
Do we only support scanMetrics in the iceberg catalog ?

Reply:
Up to now, we only support scanMetrics in iceberg catalog.